docs/components/copy.md

```diff
@@ -127,7 +127,7 @@ Tooltip text displayed on successful copy. Defaults to `Copied!`.
 
 ### monospace
 
-An indicator of whether the copyable text has `JetBrains Mono` font or not. Defaults to `true`, but `false` if `badge` is `true`.
+An indicator of whether the copyable text has `JetBrains Mono` font or not. Defaults to `undefined`, but `true` if `badge` is `false`.
```

Comment:
We want to use monospace font when `badge` prop is `false`. If `monospace` prop is `false` be default - no matter what they pass through this prop - it's always gonna be a monospace font in a non-badge.

I don't like this pattern though - to set a `boolean` prop to default to `undefined` so I change default back to `false` -- this will have to be addressed if we want to not use the monospace in a non-badge in future.